packages/graphql-eslint-rules/src/__tests__/ban-parameters.test.ts

```diff
@@ -0,0 +1,53 @@
+import { GraphQLRuleTester } from "@graphql-eslint/eslint-plugin";
+import rule from "../ban-parameters";
+
+const ruleTester = new GraphQLRuleTester();
+
+ruleTester.runGraphQLTests("ban-parameters", rule, {
+  valid: [
+    {
+      code: `
+extend type Query {
+  messageOfTheDay(date: Date!): String
+}`,
+      options: [
+        {
+          bannedParameters: [
+            {
+              keywords: ["force", "correlationId"],
```

Comment:
I know that this rule doesn't yet ban any specific param, but it will be important to make sure that we can provide helpful reporting the dev that's specific to the banned parameter in question. E.g. for `correlationId` I expect to be able to print a message like: "This parameter is used as a side-effect to track correlated work through telemetry. This should be passed as context instead, see [URL] for details on how to do so with Apollo and Relay."

Reply:
yes, config supports multiple groups so you can provide distinct descriptions for each group of keywords. I'll add test case that illustrates it.